conduit/plugins/importers/algod/algod_importer_test.go

```diff
@@ -345,15 +356,18 @@ func TestGetBlockContextCancelled(t *testing.T) {
 		{"archival", NewAlgodServer(GenesisResponder,
 			BlockResponder,
 			BlockAfterResponder,
-			MakeSyncRoundResponder(http.StatusOK))},
+			MakeSyncRoundResponder(http.StatusNotFound))},
 		{"follower", NewAlgodServer(GenesisResponder,
 			BlockResponder,
 			BlockAfterResponder, LedgerStateDeltaResponder,
 			MakeSyncRoundResponder(http.StatusOK))},
 	}
 
 	for _, ttest := range tests {
+		ttest := ttest
 		t.Run(ttest.name, func(t *testing.T) {
+			// this didn't work...
```

Comment:
Is test parallelization a blocker for PRs in this repo? It looks like it's not enforced in the repo by a linter [unlike go-algorand](https://github.com/algorand/go-algorand/blob/master/.golangci.yml#L20), but maybe it is implicitly done or a future todo.

Reply:
It is not required but maybe it should be. I added a few on in this PR since I was changing the tests already.